isisd/isis_sr.c

```diff
@@ -1669,12 +1702,38 @@ static void sr_adj_sid_del(struct sr_adjacency *sra)
 		exit(1);
 	}
 
+	if (sra->type == ISIS_SR_LAN_BACKUP) {
+		sra->backup_nexthops->del =
+			(void (*)(void *))isis_nexthop_delete;
+		list_delete(&sra->backup_nexthops);
+	}
+
```

Comment:
Need to check that sra->backup_nexthops is not null in the case you install a backup ADJ_SID without TI-LFA (in accordance to my previous remark)

Reply:
Done in preparation for the upcoming backup Adj-SID changes.